Example/test-suite/tests/Colors.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+import { waitForDetailed } from './helpers';
+import {
+  runOnUI,
+  processColor as reaProcessColor,
+} from 'react-native-reanimated';
+import { processColor } from 'react-native';
+
+export const name = 'Colors';
+
+export function test(t) {
+  t.describe('colors', () => {
+    t.it('process basic colors', async () => {
+      const testData = [
+        'red',
+        'green',
+        'blue',
+        'white',
+        'black',
+        'cyan',
+        'orange',
+        'yellow',
+        'purple',
+      ];
+
+      const uiResults = {};
+      testData.forEach((color) => {
+        // run on JS thread
+        t.expect(processColor(color)).toBe(reaProcessColor(color));
```

Comment:
Does it make a any difference if we swap processColor with reaProcessColor?

Reply:
Not at all. The only difference would probably be an order of how they're printed once the test fails, like `test failed: got aaa, expected bbb`